### PR TITLE
Javascript manipulation on hamburger

### PR DESCRIPTION
**Pull request by azeezumoru**
- My partner and I set up eslintrc.json file

- We created index.js file and linked it to the index.html

- Added event listener to the hamburger icon

### CODE REVIEW

**Comment by prolajumokeoni** [comment]
Hi @azeezumoru and @brightbrain20, You are missing some sections in your readme, please add all sections from the microverse [readme template](https://github.com/microverseinc/readme-template)  to your readme for your own application

**Comment by prolajumokeoni** [comment]
Hi @Team 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a descriptive readme,  You are missing some sections in your readme

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: styles.css [hunk 1]
```diff
@@ -80,6 +84,59 @@ ul {
...
+}
+
+#mobile-navigation {
+  position: fixed;
+  background-color: #3c3a39;
+  left: 100%;
+  top: -25px;
+  width: 100%;
+  padding: 140px 20px 0;
+  height: 100%;
```
Comment (omar-muhamad) [inline styles.css]: - [ ] please add the correct padding or margin top value by decreasing it to match the given Figma design.

| Given Design  | Your Design |
| ------------- | ------------- |
| image  | image |

File: index.html [hunk 2]
```diff
@@ -16,17 +16,27 @@
...
     <div class="dropdown">
-      <img src="./Images/Icon - Menu.png" alt="">
+      <img class="dropdown-img" src="./Images/Icon - Menu.png" alt="">
+    </div>
+    <div id="mobile-navigation">
+      <div class="remove-btn hidden">X</div>
+      <ul class="mobile-nav">
+        <li> <a href="#about-myself" class="mobile-item">Portfolio</a> </li>
+       
+        <li> <a href="#about-me" class="mobile-item">About</a></li>
```
Comment (omar-muhamad) [inline index.html:29]: - [ ] Clicking on the About mobile menu option suppose to show the `about me` section, not showing the page from the beginning.

chrome-capture-2023-0-4

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi Team,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [x] Nice job using the Microverse readme template but there are a lot of section missing, So please add these sections (Getting started, Future Features, Show your support, and License) 

- [x] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by alexander16108 — APPROVED** [review]
Hi @azeezumoru , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Reem-lab — COMMENTED** [review]
Hi @Omar-Muhamad, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that you should submit your PR from your repo since this milestone you should both work in both projects but just submit the PR indvidually but now you submit your partner PR so kindly implement this also in your repo then submit your PR

image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**